docs/pages/world/tables.mdx

```diff
@@ -20,6 +21,13 @@ When a `System` reads or writes storage via [table libraries](/store/table-libra
 
 <Illustration />
 
+<Callout type="warning" emoji="⚠️">
+  Once a table is created its schema is *immutable*.
+  If you need to add fields to an existing table, create a new table with the same key schema and the new fields in the value schema, and retrieve from both tables to get the complete value.
+  If you need to delete fields, just have your code ignore them.
+
+</Callout>
```

Comment:
can we add this to `store/tables.mdx` page too?

Reply:
Added.